tests/sdk/test_session_context.py

```
@@ -0,0 +1,625 @@
+    the module loading process to trigger line 9.
+    """
+    import typing
+
+    # Create a custom module loader that will execute the TYPE_CHECKING import
+    def execute_type_checking_import():
+        """Execute the import from line 9 to ensure coverage."""
+        # This directly mimics what line 9 does: from .peer import Peer
+        try:
+            from sdks.python.src.honcho.peer import Peer
+
+            return Peer
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)